### PR TITLE
[5.1] Move CONTRIBUTING.md and add Github templates for PR and issues

### PR DESCRIPTION
**Pull request by lucasmichot**
Fill the issues and PR description field using the following templates.
This is a starting point and needs to be edited depending on your expectations @taylorotwell 
Targeting `5.1` as these templates should be displayed on issues and PR related to the `5.1` branch too. 

---
- This can also be applied to other Laravel projects: laravel, cashier and so on
- See https://help.github.com/articles/helping-people-contribute-to-your-project/ 
- See https://github.com/symfony/symfony/blob/master/.github/PULL_REQUEST_TEMPLATE.md
- Related to #2990
- Closes #12364

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Changes proposed in this pull-request
+
+-
+-
+-
+
+---
+
+Fixes #
```
Comment (GrahamCampbell) [inline .github/PULL_REQUEST_TEMPLATE.md]: maybe include a fullstop here too

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+### Expected behaviour
```
Comment (GrahamCampbell) [inline .github/ISSUE_TEMPLATE.md]: We should definitely force people to include the full laravel version and php version here.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (lucasmichot) [inline .github/PULL_REQUEST_TEMPLATE.md]: After the `#` you mean ?

Comment (GrahamCampbell) [inline .github/PULL_REQUEST_TEMPLATE.md]: yeh

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+| Q               | A
+| --------------- | ---
+| Laravel version | 5.6.x
+| PHP version     | 5.1 / 5.2
```
Comment (vinkla) [inline .github/ISSUE_TEMPLATE.md]: Laravel and PHP needs to be swapped.

Comment (lucasmichot) [inline .github/ISSUE_TEMPLATE.md]: See https://github.com/laravel/framework/commit/a2858274a96353904fc1e81241c5dbe768825ca8

**Comment by lucasmichot** [comment]
I won't commit more to this PR, I let Taylor change it afterwards to match his needs

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (vinkla) [inline .github/ISSUE_TEMPLATE.md]: :+1:

**Comment by lucasmichot** [comment]
As a preview:
* https://github.com/lucasmichot/framework/blob/feature/5.1/templates-bis/.github/ISSUE_TEMPLATE.md
https://github.com/lucasmichot/framework/blob/feature/5.1/templates-bis/.github/PULL_REQUEST.md

**Comment by vinkla** [comment]
@lucasmichot looks good :+1:

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+| Q               | A
+| --------------- | ---
+| Laravel version | 5.1 / 5.2
+| PHP version     | 5.6.x
```
Comment (GrahamCampbell) [inline .github/ISSUE_TEMPLATE.md]: we need the PATCH version too for BOTH php and laravel, otherwise they might as well just say, "laravel 5", lol

Comment (lucasmichot) [inline .github/ISSUE_TEMPLATE.md]: LOL, yeah crap !
_Laravel version: PHP (I think the thing my developer uses finishes with a P)_

@taylorotwell will fine-tune that

**Comment by taylorotwell** [comment]
Personally I don't really have a strong desire for a PR template. Graham is in the issues more so he may have opinions on that end which can be made in a separate PR.

**Pull request closed without merging**